vaadin-testbench-unit-shared/src/main/java/com/vaadin/testbench/unit/TesterWrappers.java

```diff
@@ -326,7 +340,16 @@ default UploadTester<Upload> test(Upload upload) {
         return BaseUIUnitTest.internalWrap(UploadTester.class, upload);
     }
 
+    default <V> VirtualListTester<VirtualList<V>, V> test(VirtualList<V> virtualList) {
+        return BaseUIUnitTest.internalWrap(VirtualListTester.class, virtualList);
+    }
+
+    default <V> VirtualListTester<VirtualList<V>, V> test(VirtualList virtualList, Class<V> itemType) {
```

Comment:
minor: 
`itemType` is not used, why we need this method? Maybe we shall deprecate?
This also needs the warning suppress, because of raw use of `VirtualList`.

Reply:
The purpose of the parameter is to infer the correct generic type of the `VirualList` item.
If we remove it, the end user will be forced to cast the result in the test code, that is not nice.
The same pattern is applied for all generic components (e.g. `ComboBox`)

Reply:
Ah, I see, thanks, it's needed for `VirtualListTester<VirtualList<V>, V>` despite it's not directly used in the method's body.